project/classic/dispatcher/image/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
+rm -fr src.origin && cp -r ../src src.origin && docker build -t aemc/dispatcher-publish:latest .
```

Comment:
Since this dispatcher image implements AMS-specific rules, shouldn't we explicitly state in its name that it's ams specific? I'd suggest, `aemc/dispatcher-publish-ams`.

Reply:
renamed to `acme/aem-ams/dispatcher-publish`